airbyte-config/config-models/src/main/resources/types/SuggestedStreams.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Can we add more metadata?
like a description or a title that would indicate what use case the suggested streams are trying to achieve?  
It seems your github example exclude some streams related to workflow for example. Those can be useful to investigate CI. 
(There might be some connectors that would also benefit from multiple suggestedStream with different use case each but that might be beyond the scope here)

Reply:
For sure! I added more context in c025313a0c6.

I like the idea of context-aware suggestions for the future!  That's why this is an object.  @michel-tricot and I brainstormed a `streams-matcher` array which might one day contain the logic for database sources to deselect tables with foreign keys, or any stream coming from a non-public namespace/scheams.